### ISSUE TITLE
Processes die ever since I upgraded to 1.0.7.

### ISSUE THREAD

**Issue by ianwalter** [maxogden/monu#33]
Logs just says:

```
mon : child 983
mon : write pid to /Users/ian/Library/Application Support/Monu/data/pids/redis.pid
mon : exit(127)
```

The commands execute fine if I run them manually and this is happening on 2 of my computers. I tried restoring the old version, but got the same issue. My config file looks like:

``` json
{
  "logs": "./logs",
  "pids": "./pids",
  "processes": {
    "redis": "redis-server",
    "postgres": "postgres -D /usr/local/var/postgres"
  }
}
```

Any ideas? Thanks.

**Comment by ianwalter** [comment]
Works if I give the full path to the binary `/usr/local/bin` but not sure why I have to do that now. Especially since `/usr/local/bin` is in the $PATH if I echo it as a command.

**Comment by max-mapper** [comment]
sorry it's an upstream issue: https://github.com/sindresorhus/shell-path/issues/4

I think it was caused by electron at some point...

**Comment by sindresorhus** [comment]
The issue was fixed in `shell-path`.

**Comment by marcbachmann** [comment]
New version of shell-path landed in v1.1.0. Thanks.
